packages/image_picker/image_picker_platform_interface/lib/src/method_channel/method_channel_image_picker.dart

```diff
@@ -102,6 +102,37 @@ class MethodChannelImagePicker extends ImagePickerPlatform {
     );
   }
 
+  @override
+  Future<PickedFile> pickMedia({
+    ImageSource source,
+    RetrieveType type,
+    double imageMaxWidth,
+    double imageMaxHeight,
+    int imageQuality,
+    Duration videoMaxDuration,
+    CameraDevice preferredCameraDevice = CameraDevice.rear,
+  }) {
+    switch (type) {
```

Comment:
We should avoid these `switches` on Enum IMO, because when you add a new value to the Enum, you break [whoever was switching on it](https://github.com/dart-lang/sdk/issues/32433), if they don't have a `default` case. 

There should always be a `default` case (or possibly better, *not* switch on Enums :P)